### DISCUSSION TITLE
Disable username update

### DISCUSSION THREAD

**Discussion by Bhautik0110** [zitadel/zitadel#4830] [Q&A]
Is that any way to disable the user login name update?

**Comment by Bhautik0110** [comment]
Another question is: Is that any way to redirect the user to the application after password reset success/user verified success?

**Comment by hifabienne** [comment]
Hi @Bhautik0110 
At the moment this is not possible. I think it is something that has to do with this issue: https://github.com/zitadel/zitadel/issues/3965
But it is not high in priority right now so we don't know when it will be done.

To your second question you can define your default redirect url on the instance settings. The user will be redirected to that url if we do not have a redirect url from your authorization request. https://github.com/zitadel/zitadel/issues/3965

**Reply by eskan** [reply]
Hello @hifabienne, the issues you've mentioned are not related to the particular case of "username" that, in my opinion, should be consider as an "id" and shouldn't be updated by the user and with care by an admin. Cause it can be used in application to handle internal authorization.
